### PR TITLE
E2E test cleanup

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

**Comment by mwielgus** [comment]
cc: @piosz 

Test failure is justified.

File: integration/heapster_api_test.go [hunk 1]
```diff
@@ -236,10 +236,13 @@ func runHeapsterMetricsTest(fm kubeFramework, svc *kube_api.Service) error {
 	if err != nil {
 		return err
 	}
+	glog.Infof("Expected pods: %v", expectedPods)
```
Comment (gmarek) [inline integration/heapster_api_test.go]: V(2)

File: integration/heapster_api_test.go [hunk 2]
```diff
@@ -236,10 +236,13 @@ func runHeapsterMetricsTest(fm kubeFramework, svc *kube_api.Service) error {
...
+
 	expectedNodes, err := fm.GetNodes()
 	if err != nil {
 		return err
 	}
+	glog.Infof("Expected nodes: %v", expectedNodes)
```
Comment (gmarek) [inline integration/heapster_api_test.go]: V(2)

**Comment by gmarek** [comment]
LGTM

File: integration/heapster_api_test.go (hunk 2 above)
Comment (mwielgus) [inline integration/heapster_api_test.go]: Done

File: integration/heapster_api_test.go (hunk 1 above)
Comment (mwielgus) [inline integration/heapster_api_test.go]: Done

**Pull request merged**